bin/whisper-fill.py

```diff
@@ -23,13 +23,14 @@
 except ImportError:
     HAS_OPERATOR = False
 
-import itertools
 import time
 import sys
 import optparse
 
 if sys.version_info >= (3, 0):
     xrange = range
+else:
+    from future_builtins import filter
```

Comment:
Not 100% sure if it is needed in this specific case. But I've added it to be sure we get same behaviour in all version.